libc/include/pthread.h.def

```diff
@@ -32,6 +32,9 @@ enum {
   PTHREAD_MUTEX_ROBUST = 0x1,
 };
 
+#define PTHREAD_PROCESS_PRIVATE 0
```

Comment:
the pthread macros should be moved to a `pthread_macros.h`at some point, though that doesn't need to be this patch.

Reply:
I agree. Let me file a TODO and link to it from here.

Reply:
0bacb1b21d3c